personal.php

```diff
@@ -1 +1,2 @@
-<!-- Placeholder file -->
\ No newline at end of file
+<?php
+?>
```

Comment:
Closing tags is never a good idea in php files

Reply:
Good Point !